doc/build/windows.md

```diff
@@ -58,6 +58,9 @@ The 64-bit (x86_64) binary will only run on 64-bit Windows and will otherwise re
 
  1. [Download](https://julialang.org/downloads) the latest version of Julia.
     Extract the binary to a reasonable destination folder, e.g. `C:\julia`.
+    By default, Julia will be installed into
+    `C:\Users\YOURNAME\AppData\Local\Julia-1.0.0\bin`,
```

Comment:
Not true on master, should be `Julia VERSION` where `VERSION = 1.x`

Reply:
Will fix